cli/src/commands/jobs.rs

```diff
@@ -138,7 +138,6 @@ pub async fn handle_submission(api: &mut PhylumApi, matches: &clap::ArgMatches)
                 );
             }
 
-            request_type = res.package_type;
```

Comment:
This means that all job submissions will use the package type from your config file (which defaults to NPM). But API ignores the value we put here, so that's fine.

Reply:
Can we not submit the lockfile type here?

Reply:
Because that's not a `PackageType`... I'm not sure I understand the suggestion...

Reply:
The API doesn't check it anyway, right? And that's ultimately what type we have? So shouldn't the goal be to submit the lockfile type?

Reply:
The API doesn't do anything with the value. But it does deserialize it with serde. So it has to be present and has to be a valid `PackageType`.